src/app/modules/webhook/webhook.service.ts

```diff
@@ -88,6 +90,18 @@ export const sendWebhook = (
     epoch: now,
   })
 
+  // Generate S3 signed urls
+  const signedUrlPromises: Record<string, Promise<string>> = {}
```

Comment:
should unit tests be added in `webhook.service.spec.ts` for checking if the webhook is being sent with the expected attachment URLs?

aws can be mocked in the unit test with 

```ts
// webhook.service.spec.ts
import { aws as AwsConfig } from 'src/app/config/config'

...
jest
  .spyOn(AwsConfig.s3, 'getSignedUrlPromise')
  .mockImplementation((_operation, params) => {
    return Promise.resolve(`https://some-fake-url/${params.Key}/${params.Expires}`)
  })
```